docs/PacketFence_Clustering_Guide.asciidoc

```diff
@@ -196,9 +204,6 @@ type=internal
 ip=192.168.3.10
 type=internal
 
-----
```

Comment:
I think that if you remove those, then it becomes a big gray ugly block across multiple pages in the PDF guide.

This is where they came from

We just have to decide whether or not the PDF is still important now that the online version is the official one

Reply:
If you put this before a "level1 Titling" it shows a big bar yes, that's why I put them off.
https://172.20.120.1:1443/static/doc/PacketFence_Clustering_Guide.html#